app/views/reviews/show.html.erb

```diff
@@ -98,25 +98,11 @@
         </tr>
         <tr>
```

Comment:
It is difficult to count them in. a diff, but it looks like this <tr> isn't closed with a matching </tr>